mm2src/lp_ordermatch.rs

```diff
@@ -3550,6 +3594,7 @@ impl OrderbookP2PItem {
             base_min_volume,
             rel_max_volume,
             rel_min_volume,
+            conf_settings: None,
```

Comment:
If I understand correctly, `best_orders` RPC call doesn't return `conf_settings`. Is this information useful in `best_orders` response?
cc @artemii235 @smk762

Reply:
I think it could be

Reply:
`best_orders` is used in a simple trading view. I think it might be useful to have a possibility to display swap estimation time there. It's worth `conf_settings` to `best_orders` response too.

Reply:
Done